ecs-agent/tmds/handlers/fault/v1/handlers/handlers_test.go

```diff
@@ -1104,8 +1101,113 @@ func generateNetworkPacketLossTestCases(name, expectedHappyResponseBody string)
 	return tcs
 }
 
+func generateStartNetworkPacketLossTestCases() []networkFaultInjectionTestCase {
+	happyNetworkPacketLossReqBody := map[string]interface{}{
+		"LossPercent": lossPercent,
+		"Sources":     ipSources,
+	}
+	commonTcs := generateNetworkPacketLossHandlerTestCases("start network-packet-loss")
```

Comment:
A bit confused by the diff between `generateNetworkPacketLossHandlerTestCases` and `generateStartNetworkPacketLossTestCases`. Given this line, can we rename `generateNetworkPacketLossHandlerTestCases` -> `generateCommonNetworkPacketLossTestCases`

Reply:
+1 Probably a good idea to rename these functions. I'll also make this update in my next PR for black hole port

Reply:
Makes sense to me. Addressing in the next revision